internal/winapi/jobobject.go

```diff
@@ -80,9 +86,68 @@ type JOBOBJECT_BASIC_PROCESS_ID_LIST struct {
 	ProcessIdList             [1]uintptr
 }
 
+// AllPids returns all the process Ids in the job object.
+func (p *JOBOBJECT_BASIC_PROCESS_ID_LIST) AllPids() []uintptr {
+	return (*[(1 << 27) - 1]uintptr)(unsafe.Pointer(&p.ProcessIdList[0]))[:p.NumberOfProcessIdsInList:p.NumberOfProcessIdsInList]
```

Comment:
the `[(1 << 27) -1]` part, I'm pretty sure we have to do something similar in other places in the repo, could we be consistent with those if possible?

Reply:
Can you point to one of these? This is just a method they use in /x/sys/windows (https://github.com/golang/sys/blob/eeed37f84f13f52d35e095e8023ba65671ff86a1/windows/security_windows.go#L614)

Reply:
Hmm, the only example i could find was https://github.com/microsoft/hcsshim/blob/0ab229b358c7b9345736c545627ed1b9d114b4a5/internal/winapi/utils.go#L18 so I suppose it's not that important. Plus the value you use is much, much larger haha

Reply:
I'm going to leave it as is if thats ok